src/main/java/org/osiam/ResourceServer.java

```diff
@@ -67,13 +55,16 @@
     @Value("${org.osiam.resource-server.db.vendor}")
     private String databaseVendor;
 
-    @Bean
-    public static PropertySourcesPlaceholderConfigurer properties() {
-        PropertySourcesPlaceholderConfigurer propertySources = new PropertySourcesPlaceholderConfigurer();
-        Resource[] resources = new ClassPathResource[] { new ClassPathResource("resource-server.properties") };
-        propertySources.setLocations(resources);
-        propertySources.setIgnoreUnresolvablePlaceholders(true);
-        return propertySources;
+    public static void main(String[] args) {
+        System.setProperty("spring.jpa.hibernate.naming_strategy", "org.hibernate.cfg.ImprovedNamingStrategy");
```

Comment:
So the column name must be changed to the new name from `SpringNamingStrategy`. This is also true for all the other classes. Additionally, we must change the DB schema again, but I think we can just change the existing migration (for `ImprovedNamingStrategy`), because we haven't released it yet.